File: Ohtap E Commerce Domain Models/Development/NetFramework/Models Objects/Models Objects/PackageAttributes/PackageAttributeBasic.cs
﻿using Ohtap.ECommerce.Domain.Models.Objects.Attributes;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Ohtap.ECommerce.Domain.Models.Objects.PackageAttributes
{
    public class PackageAttributeBasic : IPackageAttributeData
    {
        public virtual string ID { get; set; }
        public virtual int Order { get; set; }
        public virtual string Description { get; set; }
        public virtual int State { get; set; }
        public virtual string Value { get; set; }
        public virtual IAttributeData Attribute { get; set; }
    }
}

File: Ohtap E Commerce Domain Models/Development/PHP/Objects/PackageAttributes/PackageAttributeBasic.php
<?php
namespace Ohtap\ECommerce\DomainModels\Objects\PackageAttributes;



class PackageAttributeBasic implements IPackageAttributeData
{   public $ID;
    public $Details;//AttributeDetailDataList
    public $Attribute;//Attribute
    public $State;
    public $Value;
    public $OrderNo;
    public $Description;

}